UnitTests/expected/Storyboards-All-CustomName.swift.out

```diff
@@ -93,6 +111,9 @@ struct XCTStoryboardsScene {
       return vc
     }
   }
+  enum TableViewCell: StoryboardSceneType {
```

Comment:
Yeah, it is a little weird since XIBs don't have "scenes". I think in this case the `TableViewCell` "scene" should just be suppressed because you can't `instantiateInitialViewController` from a XIB.